versioned_docs/version-v1.1/rancher/node/node-driver.md

```diff
@@ -31,6 +31,55 @@ When the Harvester node driver is enabled, you can create Kubernetes clusters on
 ### Support Matrix
 Refer to [Rancher Downstream Cluster Support Matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-6-9)
 
+## Known issues
+
+| Summary                                                                                                                                                                                   | Status    | Last updated |
+|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|--------------|
+| [Volumes created by the Harvester CSI driver in the host Harvester cluster would be deleted after editing/deleting the guest cluster](https://github.com/harvester/harvester/issues/3272) | Mitigated | 2023-03-15   |
+
+## Issue details
+
+### Volumes created by the Harvester CSI driver in the host Harvester cluster would be deleted after editing/deleting the guest cluster.
+| Status    | Last updated |
+|-----------|--------------|
+| Mitigated | 2023-03-15   |
+
+**Workaround**: Temporary change the Harvester node driver version to [v0.6.3](https://github.com/harvester/docker-machine-driver-harvester/releases/tag/v0.6.3) in Rancher can help. This can be accomplished via the Rancher UI.
+1. Click `View in API` in Rancher `Cluster Management` > `Drivers` > `Node Drivers` > `Harvester` > `⋮`
+2. Click `Edit`
+3. Uncheck the `builtin` checkbox
+4. Change the `*url` to `https://releases.rancher.com/harvester-node-driver/v0.6.3/docker-machine-driver-harvester-amd64.tar.gz`
+5. Change the `checksum` to `159516f8f438e9b1726418ec8608625384aba1857bc89dff4a6ff16b31357c28`
+6. Click `Show Request` > `Send Request`
+
+:::caution
+
+Changes to the node driver cannot be persisted and will be restored after the Rancher container restarts
+
+:::
+
+:::caution
+
+The Harvester node driver version v0.6.3 removed the `qemu-guest-agent` auto-injection from the backend. If you are using an image that does not contain the `qemu-guest-agent` package, you must install and start `qemu-guest-agent` using userdata, otherwise, the cluster will not be able to be provisioned successfully.
```

Comment:
```suggestion
Starting with v0.6.3, the Harvester node driver has removed the `qemu-guest-agent` auto-injection from the backend. If the image you are using does not contain the `qemu-guest-agent` package, you can use `userdata` config to install and boot `qemu-guest-agent`. Otherwise, the cluster will not be provisioned successfully.
```